src/app/central.py

```diff
@@ -1,21 +1,53 @@
 """
 Central server functions
 """
-
+import io
 import json
-import time
 import multiprocessing
+import threading
+import time
 from datetime import datetime
-from http.server import HTTPServer
+from http.server import HTTPServer, BaseHTTPRequestHandler
 import matplotlib.pyplot as plt
 
 import config
 from circularqueue import CircularQueue
-from httprequests import RequestHandler
 
 
-class CentralRequestHandler(RequestHandler):
+class CentralRequestHandler(BaseHTTPRequestHandler):
     queue = multiprocessing.Queue()
+    circular_queue = CircularQueue(100)
+
+    def do_GET(self):
```

Comment:
Central now has to implement the `GET` method since it's no more inheriting implementation from `httprequests.py` (removed).